File: openesdh-repo/src/main/java/dk/openesdh/exceptions/contacts/NoSuchContactException.java
package dk.openesdh.exceptions.contacts;

import org.alfresco.error.AlfrescoRuntimeException;

/**
 * @author Lanre Abiwon.
 */
public class NoSuchContactException extends AlfrescoRuntimeException {

    private static final long serialVersionUID = -351467831095433997L;

    public NoSuchContactException(){
        super(String.format("Contact does not exist or could not be created"));
    }
}

File: openesdh-repo/src/main/java/dk/openesdh/exceptions/contacts/NoSuchContactException.scala
package dk.openesdh.exceptions.contacts

import org.alfresco.error.AlfrescoRuntimeException

/**
 * @author Lanre Abiwon.
 */
@SerialVersionUID(-351467831095433997L)
class NoSuchContactException(errorMessage:String) extends AlfrescoRuntimeException(errorMessage) {}